app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -11,7 +11,7 @@
     <Thumbnail
       v-if="!hideThumbnail"
       :src="currentContact.thumbnail"
-      :badge="chatMetadata.channel"
+      :badge="thumbnailBadge"
       class="columns"
       :username="currentContact.name"
       :status="currentContact.availability_status"
```

Comment:
Please pass a prop 'variant' so that the logic is handled inside the thumbnail component, not here.

Reply:
Also, please use an icon for SMS inboxes.